e2etest/bgptests/bgp.go

```diff
@@ -174,6 +174,17 @@ var _ = ginkgo.Describe("BGP", func() {
 				testservice.TrafficPolicyCluster(svc)
 				testservice.ForceV6(svc)
 			}),
+		table.Entry("IPV4 - ExternalTrafficPolicyCluster - request IPv4 via custom annotation", ipfamily.IPv4, "ExternalTrafficPolicyCluster", []string{v4PoolAddresses},
+			func(svc *corev1.Service) {
+				testservice.TrafficPolicyCluster(svc)
+				testservice.WithSpecificIPs(svc, "192.168.10.100")
+			}),
+		table.Entry("DUALSTACK - ExternalTrafficPolicyCluster - request Dual Stack via custom annotation", ipfamily.DualStack, "ExternalTrafficPolicyCluster", []string{v4PoolAddresses, v6PoolAddresses},
+			func(svc *corev1.Service) {
+				testservice.TrafficPolicyCluster(svc)
+				testservice.DualStack(svc)
+				testservice.WithSpecificIPs(svc, "192.168.10.100", "fc00:f853:0ccd:e799::100")
```

Comment:
Here we don't have control if the test will actually receive the ips we are asking for.
We only rely on the existing tests, which make sure the ips assigned to the service will work.